com.unity.netcode.adapter.utp/package.json

```diff
@@ -1,11 +0,0 @@
-{
-  "name": "com.unity.netcode.adapter.utp",
```

Comment:
I think deleting this package would mean something on UDash/Operate side too.
We should update that catalog thing we maintain for online dashboards people use via Unity Cloud Services site @ashwinimurt

Reply:
yes there is, I will do a PR there and add you.